tools/lint/lint.py

```diff
@@ -116,7 +115,7 @@ def all_filesystem_paths(repo_root, subdir=None):
             if subdir:
                 path = os.path.join(subdir_str, path)
             assert not os.path.isabs(path), path
-            yield ensure_text(path)
+            yield path.decode("utf8")
```

Comment:
This is wrong on certain platforms, but perhaps we don't have to care about such files?

Reply:
Maybe file an issue on that?

Reply:
I've filed https://github.com/web-platform-tests/wpt/issues/28933.